src/Symfony/Component/PasswordHasher/Hasher/NativePasswordHasher.php

```diff
@@ -29,9 +29,9 @@ final class NativePasswordHasher implements PasswordHasherInterface
     private $options;
 
     /**
-     * @param string|null $algo An algorithm supported by password_hash() or null to use the stronger available algorithm
+     * @param string|null $algo An algorithm supported by password_hash() or null to use the best available algorithm
```

Comment:
I'm not sure I understand the changes in this file. If we consider bcrypt to be safer than argon at this moment, shouldn't we instead modify the behavior of `null` instead of modifying the default?